rasa_nlu/server.py

```diff
@@ -58,6 +59,8 @@ def decorated(*args, **kwargs):
 
 def create_app(config, component_builder=None):
     rasa_nlu_app = Flask(__name__)
+    if 'cors_origins' in config:
```

Comment:
no need for the check, it will always be set (configuration is initialized with the defaults, so the key is always present)